test/functional/utilities/constants.go

```diff
@@ -0,0 +1,8 @@
+package testutilities
```

Comment:
are these used anywhere?

Reply:
nope, I forgot to remove them when I started the other pr